src/index.json

```diff
@@ -1492,6 +1492,52 @@
                 "sha256Digest": "073d8dce3861e82f739586c652f67344f910ed853d17edea4ad86702a693b157"
             }
         ],
+        "sap-hana": [
+            {
+                "downloadUrl": "https://github.com/Azure/azure-hanaonazure-cli-extension/releases/download/0.3.8/sap_hana-0.3.8-py2.py3-none-any.whl",
+                "filename": "sap_hana-0.3.8-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.minCliCoreVersion": "2.0.46",
```

Comment:
You don't azext.maxCliCoreVersion here. That is not right.

Reply:
Done